resources/ec2-vpn-connections.go

```diff
@@ -2,15 +2,15 @@ package resources
 
 import (
 	"fmt"
+	"github.com/rebuy-de/aws-nuke/pkg/types"
```

Comment:
Please but this in a block with the other GitHub packages

Reply:
Should be okay now. I have rebased my commit to avoid having another one just for this fix.